pkg/controller/controller.go

```diff
@@ -50,42 +54,103 @@ func NewController() (*Controller, error) {
 		return nil, err
 	}
 
+	shareClient, err := shareclientv1alpha1.NewForConfig(kubeRestConfig)
+	if err != nil {
+		return nil, err
+	}
+
+	// NOTE, not specifying a namespace defaults to metav1.NamespaceAll in
+	// informers.NewSharedInformerFactoryWithOptions, but we restrict OpenShift
+	// "system" namespaces with chatty configmaps like the leaderelection related ones
+	// that are updated every few seconds
+	tweakListOptions := internalinterfaces.TweakListOptionsFunc(func(options *metav1.ListOptions) {
```

Comment:
So in my testing, this function gets called more than once.  But on prolonged intervals, and intervals which I could not line up with the relist interval.

In theory, we could have flipped the field selector string to list namespaces we are interested in based on the shares we discover, but given this was called so infrequently, I deemed that approach untenable.